web/static/js/reducers/folder.js

```diff
@@ -2,93 +2,22 @@
 import * as actions from '../actions/folder'
 
 const initialState = {
-  loading: false,
-  error: null,
-  loadingFetch: false
-}
-
-const deleteFolder = (state, action) => {
-  const newState = {...state}
-  delete newState.folders[action.id]
-  return {
-    ...newState
-  }
-}
-
-const renameFolder = (state, action) => {
-  const newState = {...state}
-  newState.folders[action.id].name = action.name
-  newState.folders[action.id].error = null
-  return {
-    ...newState,
-    error: null
-  }
-}
-
-const createFolder = (state, action) => {
-  const newState = {...state}
-  const { folder } = action
-  newState.folders = newState.folders || {}
-  newState.folders[folder.id] = folder
-  newState.folders[folder.id].error = null
-  return {
-    ...newState,
-    error: null
-  }
-}
-
-const includeError = (state, action) => {
-  const newState = {...state}
-  newState.folders[action.id].error = action.error
-  return newState
+  fetching: false,
```

Comment:
Just a detail: should we use `loading`?

Reply:
AFAIK `loading` in Surveda is meant as "we're saving something and are waiting to _load_ the updated data", whereas `fetching` is use to mean we're fetching the data.

This seems to be consistent, so I kept the naming.